### DISCUSSION TITLE
Which is the last version of Mono that support Windows XP?

### DISCUSSION THREAD

**Discussion by 2dpdlja496or43iq** [mono/mono#21643] [Q&A]
And where to download it? Thanks.

**Comment by jesstelford** [comment]
Windows XP support was dropped in this commit: https://github.com/mono/mono/pull/1723/commits/46c5bdd114f490579be1ba23ec93953752ebf744

Which looks to have first been released in Mono [mono-4.2.0.179](https://github.com/mono/mono/releases/tag/mono-4.2.0.179). The build immediately it was [4.0.5](https://www.mono-project.com/docs/about-mono/releases/4.0.5/), but that only has a macosx binary available. The last build for windows before 4.2 is [4.0.3](https://www.mono-project.com/docs/about-mono/releases/4.0.3/)
 
Therefore, **the last version of Mono that supports Windows XP is 4.0.3**. [Download](https://download.mono-project.com/archive/4.0.3/windows-installer/).
